pkg/util/azureclient/environments.go

```diff
@@ -25,6 +29,9 @@ var (
 		ActualCloudName:          "AzureCloud",
 		GenevaMonitoringEndpoint: "https://gcs.prod.monitoring.core.windows.net/",
 		AppSuffix:                "aro.azure.com",
+		AppLensEndpoint:          "https://diag-runtimehost-staging.azurefd.net/api/invoke",
```

Comment:
There is not a pattern for that today. The RP Config is only used in the `deploy` binary. Without changing anything architecturally, what that could do is plumb the RP Config into environment variables to be pulled into the RP when it inits. Otherwise we'l need to look at adding the RP Config as a parameter to other commands

Reply:
Understood - I don't think there is necessarily a problem with the way these values are being handled right now. Maybe one of the suggestions you mentioned could be implemented for all fields later on then.